server/broadcast.go

```diff
@@ -448,6 +454,7 @@ func processSegment(cxn *rtmpConnection, seg *stream.HLSSegment) ([]string, erro
 		}
 
 		// recoverable error, retry
+		glog.V(common.DEBUG).Infof("Retrying current segment mid=%v nonce=%d seqNo=%d", mid, nonce, seg.SeqNo)
```

Comment:
I would prefer to remove this log since there is already a similar log that is being updated with the manifestID in https://github.com/livepeer/go-livepeer/pull/1875.